src/contracts/core/AllocationManager.sol

```diff
@@ -292,7 +308,7 @@ contract AllocationManager is
     function setAllocationDelay(address operator, uint32 delay) external {
         if (msg.sender != address(delegation)) {
             require(_checkCanCall(operator), InvalidCaller());
-            require(delegation.isOperator(operator), OperatorNotRegistered());
+            require(delegation.isOperator(operator), InvalidOperator());
```

Comment:
Can we delete error `OperatorNotRegistered`?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/911/commits/aeb02ac411cbc1fb79350d64fcdc5c14bb77a998